【PaddleX No.1】PaddleX 文档优化

Echo-Nie opened the pull request:
## PaddleX描述
根据最新文档：[PaddleX文档3.0](https://paddlepaddle.github.io/PaddleX/3.0/) 对 `README` 和 `README_en` 文件进行对齐，删除被废弃的文档，新增更多的文档，与3.0文档进行对齐。


操作类型 | 文档路径 | 文档名称 | 变更说明
-- | -- | -- | --
✅ 新增 | 文本图像智能分析 | 文档场景信息抽取v4产线使用教程 | 新增v4版本产线教程
✅ 新增 | 🛠️ 模型产线部署 | 获取ONNX模型 | 新增模型格式转换文档
✅ 新增 | 💻 多硬件使用 | 燧原 GCU 飞桨安装教程 | 新增国产硬件适配指南
✅ 新增 | 📂 数据标注教程 | 图像分类任务模块/目标检测任务模块等10项 | 新增标准化数据标注规范
✅ 新增 | 📑 产线列表 | PaddleX产线列表(CPU/GPU/DCU/MLU/NPU/XPU/GCU) | 新增各硬件平台产线清单
✅ 新增 | 📄 模型列表 | PaddleX模型列表（含各硬件平台） | 新增各硬件平台模型适配清单
✅ 新增 | 📝 产业实践教程 | 16个应用场景教程（论文文献抽取/车牌识别/垃圾分类等） | 新增行业应用实践案例
✅ 新增 | ⌨️ 单功能模块 | PaddleX 产线命令行使用说明 | 新增命令行操作文档
✅ 新增 | ⚙️ 单功能模块 | 产线并行推理 | 新增多线程优化指南
❌ 删除 | 文本图像智能分析 | 印章文本识别产线使用教程 | 合并到公式识别模块
❌ 删除 | 🔄 计算机视觉 | 通用图像识别产线使用教程 | 与图像分类产线合并
❌ 删除 | 📝 相关说明文件 | PaddleX 时序任务模型配置文件参数说明 | 参数说明整合到通用配置文档



预览链接： [README.md](https://github.com/PaddlePaddle/PaddleX/blob/3926416be50af0df62de59fb55d12732f95682dc/README.md) ， [README_en](https://github.com/PaddlePaddle/PaddleX/blob/3926416be50af0df62de59fb55d12732f95682dc/README_en.md)


## FAQ描述
基于原文档的基础上，参考 [常见问题——PaddleX](https://paddlexdocs.readthedocs.io/en/stable/FAQ.html)，对一些常见问题进行补充。

问题编号 | 操作类型 | 问题内容 | 修改说明
-- | -- | -- | --
Q1 | 无变化 | PaddleX是什么？ | 内容完全一致，未作修改
Q2 | 无变化 | 产线与模块的关系 | 内容完全一致，未作修改
Q3 | 无变化 | wheel包安装模式选择 | 内容完全一致，未作修改
Q4 | 无变化 | PaddleX与星河零代码产线区别 | 内容完全一致，未作修改
Q5 | ✅ 新增 | 有没有适用于低配设备的小模型？ | 新增内容：<br>• 模型裁剪功能说明<br>• VOC检测任务案例
Q6 | ✅ 新增 | 如何从之前训练的模型继续训练？ | 新增内容：<br>• pretrain_weights参数使用说明
Q7 | ✅ 新增 | PaddleX保存的几种模型有什么区别？ | 新增内容：<br>• 四类模型用途说明（正常/裁剪/部署/量化）<br>• model.yml区分方法
Q8 | ✅ 新增 | 每次训练都需要重新下载预训练模型？ | 新增内容：<br>• 手动管理模型路径方法<br>• paddlex.pretrain_dir设置说明
Q9 | 无变化 | 使用过程中遇到问题如何反馈？ | 内容完全一致，未作修改


预览链接： [FAQ.md](https://github.com/PaddlePaddle/PaddleX/blob/3926416be50af0df62de59fb55d12732f95682dc/docs/FAQ.md)  ， [FAQ_en](https://github.com/PaddlePaddle/PaddleX/blob/3926416be50af0df62de59fb55d12732f95682dc/docs/FAQ.en.md) 

## VisualDL描述
1. 标题数量太多，影响用户的使用体验，而且很多小标题并非重点，显得目录结构杂而混乱，难以让用户区分Visual的核心重点。所以将一些非核心的标题改为文本加粗形式。
2. 对于文档中的每个小标题进行编号如“一、二、三、”，“1. 2. 3.”，“a. b. c. ”，使得文档结构更清晰且有条理。
2. 修改正文中的目录结构，只包含一级标题（除了 `介绍`）。
4. 对于“命令行启动VisualDL”添加示例代码。
5. 对“核心亮点”中的内容进行修改，原文档侧重于项目本身，润色后不仅体验项目特点，还从用户角度考虑。
6. 原来的VisualDL采用的是相对路径，这里改成超链接形式。

预览链接： [VisualDL](https://github.com/PaddlePaddle/PaddleX/blob/3926416be50af0df62de59fb55d12732f95682dc/docs/VisualDL.md) ， [VisualDL_en](https://github.com/PaddlePaddle/PaddleX/blob/3926416be50af0df62de59fb55d12732f95682dc/docs/VisualDL.en.md)

issue：https://github.com/PaddlePaddle/PaddleX/issues/3557


> [!NOTE] 
> 完成了FAQ模块、VisualDL模块，PaddleX文档对齐
> update 2025/5/22：9:57 

 @changdazhou @openvino-book @luotao1

Echo-Nie committed:
1. Add the documentation for VisualDL (Zh and En) 2. Modify the README.md document for PaddleX (update the En document simultaneously), mainly modifying the 'Documentation' section of this md file 3. Modify the FAQ document (Zh and En)

Echo-Nie commented:
预览链接： [README.md](https://github.com/PaddlePaddle/PaddleX/blob/3926416be50af0df62de59fb55d12732f95682dc/README.md) ， [README_en](https://github.com/PaddlePaddle/PaddleX/blob/3926416be50af0df62de59fb55d12732f95682dc/README_en.md)

预览链接： [FAQ.md](https://github.com/PaddlePaddle/PaddleX/blob/3926416be50af0df62de59fb55d12732f95682dc/docs/FAQ.md)  ， [FAQ_en](https://github.com/PaddlePaddle/PaddleX/blob/3926416be50af0df62de59fb55d12732f95682dc/docs/FAQ.en.md) 

预览链接： [VisualDL](https://github.com/PaddlePaddle/PaddleX/blob/3926416be50af0df62de59fb55d12732f95682dc/docs/VisualDL.md) ， [VisualDL_en](https://github.com/PaddlePaddle/PaddleX/blob/3926416be50af0df62de59fb55d12732f95682dc/docs/VisualDL.en.md)

@changdazhou @openvino-book @luotao1 pls review

changdazhou commented:
代码提交前有fetch最新代码，并做pre-commit不，建议将commit压缩成一个提交，保持简洁

Echo-Nie commented:
> 代码提交前有fetch最新代码

有的，如下图
![image](https://github.com/user-attachments/assets/5fcdc485-367f-48fa-853c-fee6ddf3a562)



>并做pre-commit不，建议将commit压缩成一个提交，保持简洁

是的，如下图
![image](https://github.com/user-attachments/assets/d445b625-a259-46ff-8f2b-1ab27800b659)

已经合并为一个commit

Echo-Nie committed:
Merge branch 'PaddlePaddle:release/3.0' into PaddleX_No1

cuicheng01 commented on docs/FAQ.md:
```diff
@@ -28,6 +28,43 @@ A：星河零代码产线是PaddleX 的云端载体，底层代码与PaddleX保
 
 
+## <b>Q：有没有适用于低配设备的小模型？</b>
+
```
当前paddlex还不支持量化，这块建议删除描述

Echo-Nie replied:
Done

cuicheng01 commented on docs/FAQ.md:
```diff
@@ -28,6 +28,43 @@ A：星河零代码产线是PaddleX 的云端载体，底层代码与PaddleX保
+1. **正常训练保存模型**：可用于加载预测、作为预训练模型、导出部署模型；
+2. **裁剪训练保存模型**：仅用于预测和部署，不可作为预训练模型；
+3. **导出部署模型**：用于服务端部署，不可用于预训练；
+4. **量化保存模型**：用于加速预测，不可用于预训练。
```
这里同上

Echo-Nie replied:
Done

cuicheng01 commented on docs/VisualDL.md:
```diff
@@ -0,0 +1,375 @@
+ [**English**](./VisualDL.en.md)
+
+
```
这里增加的文档，建议写到mkdocs.yml中，作为站点文档展示的一部分

Echo-Nie replied:
Done. 但是不确定是否添加正确，还请麻烦审查一下

cuicheng01 commented on docs/VisualDL.md:
```diff
@@ -0,0 +1,375 @@
+  <img src="https://raw.githubusercontent.com/PaddlePaddle/VisualDL/develop/frontend/packages/core/public/images/logo-visualdl.svg?sanitize=true" width="70%"/>
+</p>
+
+
```
文档中logo、二维码等内容可以删除掉。另外，建议说明下PaddleX中产出的log或者权重怎么使用vdl

Echo-Nie replied:
> 文档中logo、二维码等内容可以删除掉

Done

> 另外，建议说明下PaddleX中产出的log或者权重怎么使用vdl

ok, I will first study the relevant content, and then I will add related descriptions. Thx for your review.🫡

Echo-Nie committed:
1. FAQ.md&FAQ.en.md: del the content about Quant;
2. VisualDL.en.md&VisualDL.md: del the logo and QR Code
3. mkdocs.yml: add the VisualDL
4. VisualDL: ready to add the content: How to use the log or weights produced by PaddleX with vdl. I am still in the process of learning.

Echo-Nie committed:
del the Prune content in FAQ

changdazhou commented:
可能需要辛苦解一下冲突
